### PR TITLE
fix: add support for multibyte characters

### PR DESCRIPTION
**Pull request by Haroenv**
if the length of a character is more than one (here tested with 2 and 7), it should be counted as that length, but definitely shouldn't cut in the middle of that character.
This does not work as expected yet, so the test is skipped

### CODE REVIEW

**Comment by rayrutjes** [comment]
Nice @Haroenv , feel free to also submit the solution so that we don't skip the test ;)

**Comment by Haroenv** [comment]
Couldn't simply find a rule for how to get the actual characters to not split them. Thought to add this test so we know that it *should* work

**Comment by Haroenv** [comment]
The difference now is that the character count is used, and not the string length. This means that 💩  counts as one character, while before it would count as two.

**Comment by rayrutjes** [comment]
> The difference now is that the character count is used, and not the string length. This means that 💩 counts as one character, while before it would count as two.

This is not a big issue.

**Comment by Haroenv** [comment]
I think this can be merged as-is, it will no longer break 💩  or 🧀 , but will still break (possibly) 🙌🏿 or 🏃🏿‍♀️

**Comment by Haroenv** [comment]
(by the way, ZWJ is also relevant for Arabic, and multibyte strings for some Chinese letters)

File: src/index.js [hunk 1]
```diff
@@ -19,19 +19,24 @@ export default function(text, chunkSize) {
   assertIsValidChunkSize(chunkSize);
 
   const chunks = [];
-  while (text.length > chunkSize) {
-    const splitAt = text.lastIndexOf(' ', chunkSize);
+  let characters = [...text];
```
Comment (Haroenv) [inline src/index.js]: I just realised that it matters where this is supported: node 6+

If it would be Array.from() it would be node 4+

Comment (rayrutjes) [reply src/index.js]: Fine for me.

**Comment by rayrutjes** [comment]
We are using conventional commit format to generate changelog.
Could you rebase to add the correct `fix: add support for multibyte characters`?

**Comment by Haroenv** [comment]
I usually squash on merge, but if you prefer to do it this way, it's also fine for me

**Comment by Haroenv** [comment]
Runes is amazing, it works exactly how I want it!

https://github.com/dotcypress/runes

**Review by rayrutjes — APPROVED** [review]
Impressive man! Go for it 🎉 
Make sure you give it a "conventional" commit name.

**Comment by Haroenv** [comment]
There are some more advanced bugs, but I issued them to runes itself, will see what I can do to fix it. For now it's already going well

**Pull request merged**